### DISCUSSION TITLE
Elastalert2 High Available

### DISCUSSION THREAD

**Discussion by wenhuiyu0218** [jertel/elastalert2#544] [Q&A] [answered]
Hi:

I may be use it in production env, but I didn't find any high availability solution in the documentation,relying on a single instance of ElastAlert is not really good enough in my opinion.

Thanks and kind regards,
Vincent.

**Comment by jertel** [comment, marked as answer]
Monitor the monitor.

If you setup a "Deadman's Switch" rule that routinely pings an external service then you can have the external service send out its own alert telling you that something is wrong. I've used this setup for several years and it's very reliable and has the advantage of being able to tell you when your entire Elastic cluster is down. Multiple Elastalerts can't do that.

Also, in the future please read the project guidelines. This should not have been opened as an issue. Open discussions when requesting support.

Moving this issue into a new discussion.
